src/SourceBuild/patches/winforms/0001-Condition-out-the-native-tests-in-VMR.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "Nikola Milosavljevic (CLR) false" <nikolam@microsoft.com>
+Date: Tue, 25 Mar 2025 07:52:19 -0700
+Subject: [PATCH] Condition out the native tests in VMR
+
+Backport: https://github.com/dotnet/winforms/issues/13186
```

Comment:
```suggestion
Backport: https://github.com/dotnet/winforms/pull/13187
```

Reply:
Yeah, I've just pushed the change that updates this.